tests/test_bingx_api.py

```
@@ -2,159 +2,155 @@
+            key = data.get("BINGX_KEY") or data.get("bingx_key")
+            secret = data.get("BINGX_SECRET") or data.get("bingx_secret")
+
         if key:
-            os.environ.setdefault('BINGX_KEY', key)
+            os.environ.setdefault("BINGX_KEY", key)
         if secret:
-            os.environ.setdefault('BINGX_SECRET', secret)
-        return bool(os.getenv('BINGX_KEY'))
+            os.environ.setdefault("BINGX_SECRET", secret)
+        return bool(os.getenv("BINGX_KEY"))
     except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)